packages/playwright/src/plugins/webServerPlugin.ts

```diff
@@ -102,14 +103,33 @@ export class WebServerPlugin implements TestRunnerPlugin {
       cwd: this._options.cwd,
       stdio: 'stdin',
       shell: true,
-      // Reject to indicate that we cannot close the web server gracefully
-      // and should fallback to non-graceful shutdown.
-      attemptToGracefullyClose: () => Promise.reject(),
+      attemptToGracefullyClose: async () => {
+        if (process.platform === 'win32')
+          throw new Error('Graceful shutdown is not supported on Windows');
```

Comment:
Windows users will now always see  `...<gracefully close start>` followed by `<will force kill>` in the logs. Not a big deal, but this was not the case before as we never announced graceful shutdown there. Maybe keep using `kill` on Windows ?

Reply:
Don't think it makes any observable difference because `log` is a no-op anyways (see L128). So it's purely a code complexity question. If we assigned `_killProcess` depending on the platform, I worry that'd make us split execution more, and make this slightly harder to reason about I think. So i'll keep it as is for now.